src/config/config.py

```diff
@@ -62,6 +63,7 @@ def load_downloader_config(args) -> None:
     Config.min_stars = args.get("min_stars", MIN_STARS_DEFAULT)
     Config.max_stars = args.get("max_stars")
     Config.org_name = args.get("org_name")
+    Config.repo_name = args.get("repo_name")
```

Comment:
Please declare this variable at the end of the config file one line after `org_name: list[str] = []`